cpp/llm_chat.cc

```diff
@@ -986,18 +1007,34 @@ class LLMChatModule : public ModuleNode {
           [this, sptr_to_self](TVMArgs args, TVMRetValue* rv) { GetChat()->Evaluate(); });
     } else if (name == "prefill") {
       return PackedFunc([this, sptr_to_self](TVMArgs args, TVMRetValue* rv) {
-        ICHECK_EQ(args.size(), 1);
-        GetChat()->PrefillStep(args[0]);
+        ICHECK(1 <= args.size() && args.size() <= 3);
+        if (args.size() == 1) {
```

Comment:
Document the behavior after each `if (args.size() == *)` branch because we have no docstring for PackedFunc's and it's hard to debug.

Reply:
done.